docs/docs/overview/quickstart/get-started-with-python.md

```diff
@@ -0,0 +1,198 @@
+
+![subscriber_id.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1688331839/Screenshot_2023-07-03_at_0.02.53_jmkhi3.png)
+
+Now, let's create a subscriber on Novu. Copy and paste the following code to do so:
+
+```python
+# Create a subscriber
+
+from novu.api.subscriber import SubscriberApi
+from novu.dto.subscriber import SubscriberDto
+   
+your_subscriber_id = "123"; # Replace this with a unique user ID that matches your database.
+
+subscriber = SubscriberApi("https://api.novu.co", "b23c55928da9e76702b74921a7fffee2")
+subscriber.create(SubscriberDto(
+        subscriber_id=your_subscriber_id,
+        email="abc@gmail.com",
+        first_name="John",
+        last_name="Doe"
+    )
+)
+```
+
+Run the code in your terminal like so:
+
+```bash
+python main.py # replace main.py with your file name
+```
+
+You should see the subscriber on your Novu dashboard.
+
+## Update A Subscriber
+
+To update the Subscriber details you can follow you can call the `put` method from `SubcriberApi`. Here is an example:
+
+```python
+# Update subscriber detail
+
+from novu.api.subscriber import SubscriberApi
+from novu.dto.subscriber import SubscriberDto
+subscriber = SubscriberApi("https://api.novu.co", "<YOUR_NOVU_API_KEY>")
+subscriber.put(subscriber=SubscriberDto(
+        subscriber_id="<You want to update>",
+        email="abc@gmail.com",
+        first_name="John",
+        last_name="Doe"
+))
+```
+
+Other valid fields that can be updated are `email`, `first_name` etc.
+
+<aside>
+Note: To make all of your app users subscribers, you need to programmatically add them to Novu.
+
+</aside>
+
+## Trigger A Notification
+
+Copy and paste the following code into your app to trigger a notification:
+
+```python
+from novu.config import NovuConfig
+from novu.api import EventApi
+
+NovuConfig().configure("https://api.novu.co", "<NOVU_API_TOKEN>")
+
+EventApi().trigger(
+     name="<YOUR_WORKFLOW_TRIGGER_ID>",  # The workflow ID is the slug of the workflow name. It can be found on the workflow page.
+    recipients="<YOUR_SUBSCRIBER_ID>",
+    payload={},  # Your Novu payload goes here
+)
+```
+
+Before running the code, make sure you understand the following:
+
+- The value of `notificationWorkflowId` should be the notification workflow's trigger ID/slug.
+
+![trigger_id.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776585/trigger_id_xkhsx7.png)
+
+- The value of `payload` is an array of the data that you want to be dynamically injected into the notification template content.
+- The value of `subscriberId` is the id of the subscriber on Novu. Replace `7789` with your subscriber ID.
+
+Run the code to trigger a notification!
+
+```bash
+python main.py # replace main.py with your file name
+```
+
+## Next Steps
+
+Great job! If you've reached this point, you should now have successfully Set up a channel provider, Create a notification workflow, Create a Subscriber, Update a Subscriber, Trigger a Notification in your application.
```

Comment:
Great job! If you've reached this point, you should now have successfully Set up a channel provider, Created a notification workflow, Created a Subscriber, Updatde a Subscriber, Triggered a Notification in your application.